opba-db/src/main/java/de/adorsys/opba/db/domain/entity/BankProfile.java

```diff
@@ -78,6 +80,8 @@
     private String externalInterfaces;
     private String protocolType;
     private boolean isSandbox;
+    @Convert(converter = ResultContentTypeConverter.class)
```

Comment:
Please add blank line (ensure there is blank line between annotated field and other fields)